sdk-springboot/src/main/java/io/dapr/springboot/DaprController.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+package io.dapr.springboot;
+
+import com.fasterxml.jackson.core.JsonFactory;
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.dapr.actors.runtime.ActorRuntime;
+import org.springframework.web.bind.annotation.*;
+import reactor.core.publisher.Mono;
+
+import java.io.IOException;
+import java.io.StringWriter;
+import java.io.Writer;
+import java.nio.charset.StandardCharsets;
+
+/**
+ * SpringBoot Controller to handle callback APIs for Dapr.
+ * TODO: use POJOs instead of String when possible.
+ * TODO: JavaDocs.
+ */
+@RestController
+public class DaprController {
+
+  private static final JsonFactory JSON_FACTORY = new JsonFactory();
+
+  private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();

```

Comment:
You should probably autowire it in the constructor and only create a new instance if one is not available. Also, having it being managed would allow user's to immediately provide their own version or add additional configurations

Reply:
We are discussing the serialization thing here. Thanks for the feedback. What is the advantage of using autowire?